developer-docs/how-to-guide/How_to_upload_existing_content_using_sunbird.md

```diff
@@ -10,285 +10,173 @@ allowSearch: true
+#### Form Data Parameters ####
+
+| Form Data Parameter | Parameter Type | Description                                          | Sample Value | Is Mandatory |
+|---------------------|----------------|------------------------------------------------------|--------------|--------------|
+| file                | String         | The name of the content file that you want to upload | sample.pdf   | Yes          |
 
 
 #### Response Body (Upload Content API)
 
+    ```
     {
-   
        "id": "xyz.learning.content.upload",
-       
        "ver": "3.0",
-       
        "ts": "2018-09-05T10:55:22ZZ",
-       
        "params": {
-       
            "resmsgid": "d6dui5d1-60u8-4a79-23f9-78796195c2h7",
-           
            "msgid": null,
-           
            "err": null,
-           
            "status": "successful",
-           
            "errmsg": null
        },
-       
        "responseCode": "OK",
-       
        "result": {
            "content_url": "https://xyz-public-prod.s3-ap-south-1.amazonaws.com/content/do_112534095400542340156583/artifact/highq_153614492126723.pdf",
-
            "node_id": "do_112534095400542340156583",
-           
            "versionKey": "153614492126723"
-
        }
-
     }
+    ```
 
-For the youtube content or file url, pass it as file Url in <b>Upload API</b> or set the artifact Url of the content using  [Update Content API**](http://www.sunbird.org/apis/content/#operation/Update%20Content)
-
-After uploading the content, publish the content using **[Publish Content API**](http://www.sunbird.org/apis/content/#operation/Publish%20Content). 
-
-#### Request Body (Publish Content API)
-
-    {
-
-     "request": {
+For the YouTube content or file url, pass it as a `fileUrl` in [Upload API](apis/content/#operation/Upload%20Content) or set the artifactUrl of the content using [Update Content API](apis/content/#operation/Update%20Content)
 
-     "content": {
 
-       "lastPublishedBy": "sunbird",
+### Publishing Content using API #### 
 
-       "publishChecklist": [
-           "No Hate speech, Abuse, Violence,Profanity"
-           "No Vulgarity",
-           "No Discrimination or Defamation",
-           "Is suitable for children",
-           "Appropriate Title, Description",
-           "Correct Board, Grade, Subject, Medium",
-           "Appropriate tags such as Resource Type, Concepts",
-           "Relevant Keywords",
-           "Content plays correctly",
-           "Can see the content clearly on Desktop and App",
-           "Audio (if any) is clear and easy to understand",
-           "No Spelling mistakes in the text",
-           "Language is simple to understand"
-       ]
+After uploading the content, publish the content using [Publish Content API](apis/content/#operation/Publish%20Content). 
 
-      }
-
-     }
+#### Request Body
 
+    ```
+    {
+        "request": {
+            "content": {
+                "lastPublishedBy": "sunbird",
+                "publishChecklist": [
```

Comment:
Please explain how these values are connected to the scenario outlined.

Reply:
The scenario talks of 2 instances - 1) where original content is being uploaded. 2) where existing content is uploaded. The values provide an insight to original content.

Reply:
Yes, I can understand that, however, it is not clear to a novice reader. Please explain that for clarity.

Reply:
the note that is added should suffice, in my opinion.